user-impl/src/main/java/com/example/auction/user/impl/UserServiceImpl.java

```diff
@@ -45,14 +45,34 @@ public UserServiceImpl(PersistentEntityRegistry registry, ActorSystem system, Ma
     }
 
     @Override
+    public ServiceCall<Auth, Auth> updateAuth() {
+        return auth -> {
+            return authRef(auth.getUsername()).ask(new AuthCommand.UpdateAuth(auth.getId(), auth.getUsername(), auth.getPassword()));
+        };
+    }
+
+    @Override
     public ServiceCall<NotUsed, User> getUser(UUID userId) {
         return req -> {
             return entityRef(userId).ask(UserCommand.GetUser.INSTANCE)
                     .thenApply(maybeUser ->
                             maybeUser.orElseGet(() -> {
                                 throw new NotFound("User " + userId + " not found");
                             })
-            );
+                    );
+        };
+    }
+
+    @Override
+    public ServiceCall<Auth, Auth> login() {
```

Comment:
I think the return type here should be a `Token` or a `Boolean`  or a `Done`. See commend above.